src/Azure.Functions.Cli/Actions/AzureActions/BaseAzureAction.cs

```diff
@@ -16,15 +14,6 @@ public BaseAzureAction(IArmManager armManager)
         }
 
         public async Task Initialize()
-        {
-            try
-            {
-                await _armManager.Initialize();
```

Comment:
the reason for having this is to prompt to login if the user never actually called `func azure login`, or if the cached token has expired. otherwise everytime you run any other command it'll fail until you run `login` again.